### PR TITLE
MAISTRA-2704 Print version when istiod starts

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: pilot/cmd/pilot-discovery/main.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ var (
 			return nil
 		},
 		RunE: func(c *cobra.Command, args []string) error {
+			log.Info(version.Info.LongForm())
```
Comment (dgn) [inline pilot/cmd/pilot-discovery/main.go:65]: nit: maybe prefix with `istiod `

**Review by dgn — APPROVED** [review]
approving as there's a hold, find to merge this when 2.1 is out of code freeze

**Comment by dgn** [comment]
@luksa I think this would be great to also push upstream. Can't imagine that anybody would not agree with this change

**Review by jwendell — APPROVED** [review]

**Pull request merged**